packages/prettier-plugin-java/test/empty_statement/_input.java

```diff
@@ -1,5 +1,27 @@
 public class EmptyStament {
 
+  class EmptyStament2 {
+    ;
+  }
+
+  public void emptyStatementWithoutComment() {
+    ;;;
+  }
+
+  public void emptyStatementWithComment() {
+    ;;//EmptyStatement
+  }
+
+  public void simpleForWithEmptyStatement() {
+    for (;;);
+
+    for (;;)/*test*/;
+
+    for (;;);/*test*/
```

Comment:
worth checking edge case of comments _between_ the for loop's semi colons?

Reply:
These semi colons are not considered as empty statements, so that's fine, IMO. But I can add some, that can not harm !